components/badge/ScrollNumber.tsx

```diff
@@ -116,8 +117,12 @@ export default class ScrollNumber extends Component<ScrollNumberProps, ScrollNum
   }
 
   renderNumberElement() {
+    const props = this.props;
     const state = this.state;
-    if (!state.count || isNaN(state.count as number)) {
+    if(props.displayComponent){
```

Comment:
code style is a mess...

Reply:
What exactly should I change here? It was already written this way.
This is my first time, pls give a few pointers.

Reply:
```diff
- if(props.displayComponent){
+ if (props.displayComponent) {
```